blueprint/core/blueprint_View.cpp

```diff
@@ -165,47 +165,23 @@ namespace blueprint
 
     void View::paint (juce::Graphics& g)
     {
-        if (props.contains("border-path"))
+        //TODO: Feels like this can probably be smarter.
+        if (props.contains("border-svg") && props["border-svg"].toString().isNotEmpty())
         {
-            juce::Path p = juce::Drawable::parseSVGPath(props["border-path"].toString());
+            const juce::String source = props["border-svg"].toString();
 
-            if (props.contains("border-color"))
-            {
-                juce::Colour c = juce::Colour::fromString(props["border-color"].toString());
-                float borderWidth = props.getWithDefault("border-width", 1.0);
-
-                g.setColour(c);
-                g.strokePath(p, juce::PathStrokeType(borderWidth));
-            }
+            auto drawable = juce::Drawable::createFromImageData(
+                    source.toRawUTF8(),
+                    source.getNumBytesAsUTF8());
 
-            g.reduceClipRegion(p);
```

Comment:
I think we still want this `reduceClipRegion` to make sure background fills clip to the border path, right?

Reply:
Sort of answered this in the statement about background being handled in SVG clipping path I think.

Reply:
Think I've covered this in comment above.